Lombiq.OrchardCoreApiClient/Interfaces/IOrchardCoreApi.cs

```diff
@@ -67,4 +68,33 @@ public interface IOrchardCoreApi
     [Post("/api/tenants/enable/{tenantName}")]
     [Headers(AuthorizationBearer, RequestedWithXmlHttpRequest)]
     Task<ApiResponse<string>> EnableAsync(string tenantName);
+
+    /// <summary>
```

Comment:
Let's not continue with the same interface, since this is now mixing content and tenant operations. Rather, please refactor:

- `IOrchardCoreApi` should remain a base interface, to be still used in `ApiClient<TApi>`.
- Create new derived interfaces for the tenants and content APIs.
- Rename `ApiClient` to `TenantsApiClient` and similarly create a `ContentApiClient`. Move `CreateAndSetupTenantAsync` to the former.